requirements/customer/glossary.tex

```diff
@@ -0,0 +1,83 @@
+\newglossaryentry{ID}
+{
+  name=ID,
+  plural=IDs,
+  description={
+    An ID is a string of digits, english letters (both lower- and uppercase) and slashes '-'.
```

Comment:
english -> English

Reply:
We should add that an ID must start with a letter.

Reply:
> We should add that an ID must start with a letter.

I have nothing against, but why?

Reply:
I was thinking we should add this rule because it's the standard for identifiers in programming languages.
But now that I think about it again I think it should be fine even if IDs start with numbers or dashes.